src/gotcha.c

```diff
@@ -361,3 +361,6 @@ GOTCHA_EXPORT enum gotcha_error_t gotcha_get_priority(const char* tool_name, int
   return get_configuration_value(tool_name, GOTCHA_PRIORITY, priority);
 }
 
+GOTCHA_EXPORT void* gotcha_get_wrappee(gotcha_wrappee_handle_t handle){
+  return handle;
+}
```

Comment:
As you mentioned above, this is doing a soft API break.  Old code will continue working for now, but could break in the future when we change the implementation.  

I'd suggest a hard API break, perhaps in some what that triggers compilation errors.  Let's force people to do the right thing now, which will keep bad code from propagating.